packages/lesmis/src/api/queries/useVitalsData.js

```diff
@@ -4,7 +4,7 @@
  *
  */
 import { useQuery } from 'react-query';
-import { utcMoment } from '@tupaia/tsutils';
+import { utcMoment } from '@tupaia/utils';
```

Comment:
Using `tsutils` in `lesmis` caused builds to fail.

I had a look and `tsutils` is not used in any frontend package. This does ring a bell about not being able to use  any tupaia TS package in the frontend currently.

This can be investigated and fixed at a later stage, skipping for now